src/components/Forms/CreatePostForm/__tests__/CreatePostForm.js

```diff
@@ -0,0 +1,175 @@
+import { render, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import React from 'react';
+import { CreatePostForm } from '../CreatePostForm';
+import { serviceRequest } from '../../../../services/serviceRequest';
+
+jest.mock('../../../../services/serviceRequest');
+jest.setTimeout(5000);
+
+describe('CreatePostForm test suite', () => {
+  const successPayLoad = {
+    status: 'success',
+    data: {
+      title: 'test title',
+      label: 'test label',
+      content: 'test content',
+    },
+  };
+
+  const failPayLoad = {
+    status: 'error',
+    data: {
+      title: 'test title',
+      label: '',
+      content: 'test content',
+    },
+  };
+
+  beforeAll(() => {
+    // Silence console.error
+    // eslint-disable-next-line no-console
+    console.error = jest.fn();
```

Comment:
if you are not using this mock, I don't think it is necessary given that we mock the implementation of services

Reply:
> if you are not using this mock, I don't think it is necessary given that we mock the implementation of services

This mock is being used to silence console.error. @JiejayLan